msdk-id-sirius/pom.xml

```diff
@@ -123,7 +123,11 @@
       <artifactId>cdk-silent</artifactId>
       <version>${cdk.version}</version>
     </dependency>
-
+    <dependency>
```

Comment:
This dependency is already specified in the parent pom, you don't need it.

Reply:
cdk-silent dependency exists only in msdk-io-mztab & msdk-isotopes-isotopepattern, not in parent pom
The code uses a SilentChemObjectBuilder, which can be found only in cdk-silent package